ivy/functional/ivy/general.py

```diff
@@ -3186,6 +3186,23 @@ def get_num_dims(
     ret
         Shape of the array
 
```

Comment:
you should also add the following sentence here

Both the description and the type hints above assumes an array input for simplicity,
but this function is \*nestable\*, and therefore also accepts :class:\`ivy.Container\`
instances in place of any of the arguments.

Reply:
Added the aforementioned, Should i also add _container_ and _mixed_ examples within 'functional/ivy/general.py'?

Reply:
yes go ahead and then we can merge

Reply:
changed